migtests/tests/import-file/run-import-file-test

```diff
@@ -239,6 +239,14 @@ main() {
 	step "Import data file from GCS (csv): t1_quote_escape_char1.csv -> gcs_quote_escape_char1"
 	import_data_file --data-dir "gs://voyager-automation-data" --format csv --delimiter '|' \
 			--file-table-map "t1_quote_escape_char1.csv:gcs_quote_escape_char1" --quote-char="'" --escape-char="'"
+	
+	
+	if [ "${RUN_LARGE_IMPORT_DATA_FILE_TEST}" = true ] ; then
```

Comment:
should we have time check for this to validate that ther eis no regression in performance, if nto check with a fixed expected value, maybe just print the time taken for this import. Whoever is running the pipeline can check it manually maybe and see if there is not any regression? 🤔